index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="CSS/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <div class="wrapper">
+        <header class="app-header">
+            <img class="graphic1" src="img/image_geometry_1.svg" alt="Graphic art">
+            <img class="menu-icon" src="img/ic_menu.svg" alt="Menu Icon">
+        </header>
+    
+        <div class="content">
+            <div class="name">
+                <p class="first name">Oshoke<br> Okolo</p>
```

Comment:
- [ ] It would be nice if you make this a heading(h1) tag as this will serve as the main focus for this section. It's a good convention to make your first main heading a `h1` tag for accessibility purposes and all subsequent headings can be of h2, h3 etc...